src/OrchardCore.Modules/OrchardCore.Media/Services/MediaOptionsConfiguration.cs

```diff
@@ -61,6 +61,9 @@ public class MediaOptionsConfiguration : IConfigureOptions<MediaOptions>
         // style-src applied to allow browser behaviour of wrapping raw images in a styled img element.
         private const string DefaultContentSecurityPolicy = "default-src 'self'; style-src 'unsafe-inline'";
 
+        public static int DefaultMaxUploadChunkSize = 104_857_600; // 100MB
+        public static TimeSpan DefaultTemporaryFileLifeTime = TimeSpan.FromHours(1);
```

Comment:
For these 2 static fields we get a CA2211: Non-constant fields should not be visible.
Can be fixed by making the first one a `const` and by making the 2nd one `readonly`.